code/controllers/subsystems/chemistry.dm

```diff
@@ -52,7 +52,7 @@ var/datum/controller/subsystem/chemistry/SSchemistry
 			chemical_reagents[R.id].specific_heat = final_heat
 			return final_heat
 
-	if(R.fallback_specific_heat)
+	if(R.fallback_specific_heat > 0)
```

Comment:
I don't think it is necessary as any integeter of 1 and above will return true

Reply:
This is actually false and the cause of the issue.

if(-1) returns true. Check discord for proof.